checker/tests/signedness/LocalVarDefaults.java

```diff
@@ -8,7 +8,7 @@ void methodInt(@Unsigned int unsignedInt, @Signed int signedInt) {
     int local2 = signedInt;
   }
 
-  // :: error: (annotations.on.use) :: error:  (anno.on.irrelevant)
```

Comment:
It would be nice to retain this error if possible, only for floating-point types.
A user might think that `@Unsigned` and `@Signed` are applicable to floating-point types, but they are not, and the error would prevent a possible user confusion.